.prettierignore

```diff
@@ -0,0 +1 @@
+/website
```

Comment:
Isn't the website in a separate repo now anyway? And in general, it seems good practice to base `.prettierignore` on `.gitignore`. For now, we could do something like [this](https://stackoverflow.com/questions/65635648/how-can-i-base-a-prettierignore-file-on-gitignore) to start, and if we need to add additional files to `.prettierignore` we can write a build script that copies over `.gitignore` into it.

Reply:
It actually already loads `.gitignore` and uses that by default. `Prettierignore` is added on top of that.

Reply:
I'm leaving website out of formatting because it errors when prettier runs, and since we're going to be rewriting it anyway, it's not worth the struggle.

Reply:
What do you think of removing this file after you formatted all the code if it only contains `website`?

Reply:
> What do you think of removing this file after you formatted all the code if it only contains website?

Without this file, if you run `npx prettier . --write`, then prettier will have an error.  I wouldn't remove it until we removed the `/website` folder

Reply:
Working through the dependent PR, It dawned on me that we should also ignore `src/plugins`: we have already moved them to a separate repo. Sorry for not mentioning it before. 🙈

Reply:
Yeah I can add ignore for plugins too. Plugins just worked so it never took any time to format them.